src/lib/wp-cli-process.ts

```diff
@@ -54,15 +55,25 @@ export default class WpCliProcess {
 
 	async execute(
 		args: string[],
-		{ phpVersion }: { phpVersion?: string } = {}
+		{
+			phpVersion,
+			longRunning = false,
+		}: {
+			phpVersion?: string;
+			longRunning?: boolean;
```

Comment:
What do you think if we pass the exact timeout to wait as a parameter? That way we could be more precise with the different type of executions.
Instead of having a boolean longRunning, we could have an integer with the timeout ms. By default it would be DEFAULT_RESPONSE_TIMEOUT.

Reply:
Yep, that's probably better. But I am thinking if we should just infer the timeout based on the command we are running. So import/export command has higher timeout on their own without having to explicitly call it that way.

Reply:
@sejas I removed the additional arg I added and now it uses the higher timeout when it detects a sqlite import/export command. Please re-review the PR.